lib/CL/CMakeLists.txt

```diff
@@ -302,6 +302,10 @@ add_library("${POCL_LIBRARY_NAME}" ${LIBPOCL_OBJS})
 
 harden("${POCL_LIBRARY_NAME}")
 
+if(UNIX AND (NOT APPLE))
+  target_link_options("${POCL_LIBRARY_NAME}" PRIVATE "LINKER:--exclude-libs,ALL")
```

Comment:
Did you check that we get a nice set of (PoCL-only) symbols exported with this in libpocl.so even when static LLVM linkage is done?

Reply:
yep i checked with `nm --extern-only libpocl.so | grep -i llvm` it now lists a much shorter list of symbols, and they are pocl's LLVM function names/types which have POCL_EXPORT.